enumeratum-test/src/main/scala/enumeratum/values/MacroEval.scala

```diff
@@ -0,0 +1,14 @@
+package enumeratum.values
+
+sealed abstract class MacroEval(val value: Int) extends IntEnumEntry
```

Comment:
The `enumeratum-test` code is used to declare models that might be reused later in other subprojects (e.g. in integration libs).

Since this isn't be used for that purpose, it might be a good idea to move it over to `enumeratum-core` and write a few tests to make sure the `value`s of each enum member get passed/evaluated properly, e.g.

```scala
MacroEval.Zero.value shouldBe 0
MacroEval.One.value shouldBe 1
MacroEval.Two.value shouldBe 2
MacroEval.Eleven.value shouldBe 11
```